ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -244,15 +232,32 @@ void PowerSystemNode::injectFault (const string& fault_name,
 
   if (!fault_activated && fault_enabled)
   {
-    ROS_INFO_STREAM(fault_name << " activated!");
-    // DEPRECATED
-    //index = 0;
+    // Only continue if no other faults have been injected.
+    ROS_INFO_STREAM("Attempting " << fault_name << " activation...");
+    if (CUSTOM_FAULT_ACTIVATED)
+    {
+      if (!warning_displayed)
+      {
+        ROS_WARN_STREAM("Cannot activate " << fault_name << ": custom fault already used this simulation.");
+        warning_displayed = true;
+      }
+    }
+    else
+    {
+      ROS_INFO_STREAM(fault_name << " activated!");
+      //ROS_WARN_STREAM_ONCE("Note that high power draw cannot be combined with custom faults.");
+      // TEMP: Uncomment this and the other 'TEMP' line code to re-enable restricting power faults:
+      // When uncommented, the user will only be able to use either high power draw or the custom
+      // fault feature, never both in a single simulation. Behavior of both at once is unpredictable.
+      //HPD_FAULT_ACTIVATED = true;
```

Comment:
These comments can be deleted.  I'm not envisioning a need to bring restrictions back, especially since the GUI cannot enforce them.